content/en/serverless/custom_metrics/_index.md

```diff
@@ -3,56 +3,95 @@ title: Custom Metrics
 
-Install the Datadog Lambda Library to collect and send custom metrics. Metrics sent from the Datadog Lambda Library are automatically aggregated into [distributions][3], so you can graph the `avg`, `sum`, `max`, `min`, and `count`. You can also calculate aggregations over a set of tags for the 50th, 75th, 95th, and 99th percentile values on the [Distribution Metrics][3] page.
+1. Follow the general [serverless installation instructions][5] to install the Datadog Lambda Library and configure your Lambda function. Note, you do _NOT_ need to add the Datadog Lambda Extension or Forwarder.
+1. Ensure the environment variable `DD_FLUSH_TO_LOG` is set to `false`, which instructs the Datadog Lambda Library to send metrics synchronously to the Datadog API.
+1. Ensure the environment variable `DD_TRACE_ENABLED` is set to `false`, which instructs the Datadog Lambda Library to NOT generate traces for the Lambda function.
+1. Ensure the environment variable `DD_API_KEY` (or an equivalent one when encrypted) is set with a valid Datadog API key.
+1. Import and use the helper function from the Datadog Lambda Library, such as `lambda_metric` or `sendDistributionMetric`, to submit your custom metrics following the [sample code](#custom-metrics-sample-code).
+### Running in a VPC
 
-Distribution metrics are designed to instrument logical objects, like services, independent of the underlying hosts. So, they are well-suited for serverless infrastructure because they aggregate metrics server-side instead of locally with an Agent.
+The Datadog Lambda Library requires [access to the public internet][6] to submit custom metrics **synchronously**. If your Lambda function is associated with a VPC, ensure that your function can reach the public internet.
 
-### Synchronous vs. asynchronous custom metrics
+## Enabling asynchronous custom metrics
 
-The Datadog Lambda Library supports submitting custom metrics in Lambda, both synchronously and asynchronously.
+If you want to generate custom metrics from a performance-critical Lambda function, Datadog encourage you to use Datadog Lambda Library to submit custom metrics asynchronously. The Datadog Lambda Library can either
 
-**Synchronous**: The default behavior. This method submits your custom metrics to Datadog via HTTP periodically (every 10 seconds) and at the end of your Lambda invocation. If the invocation lasts for less than 10 seconds, your custom metrics are submitted at the end of the invocation.
+- Publishes your custom metrics to the [Datadog Lambda Extension][7], which will submit your custom metrics directly to Datadog without impacting your function latency. This solution is easier to set up, but it currently only work for the popular runtimes, and does not support submitting metrics with timestamps from the past.
```

Comment:
```suggestion
- Publish your custom metrics to the [Datadog Lambda Extension][7], which submits your custom metrics directly to Datadog without impacting your function latency. This solution is easier to set up, but it currently only works for the popular runtimes, and does not support submitting metrics with timestamps from the past.
```
we should be more specific about what you mean by the popular runtimes

Reply:
Tried to be vague, because I know that list will quickly change in the future, and almost certain that we may forget updating the list here.